scripts/rollup/build.js

```diff
@@ -194,13 +194,21 @@ function getFilename(name, hasteName, bundleType) {
   }
 }
 
-function uglifyConfig({
-  mangle,
-  manglePropertiesOnProd,
-  preserveVersionHeader,
-  removeComments,
-  headerSanityCheck,
-}) {
+/**
+ * @param configs {
```

Comment:
I'm okay with this but we generally are bad at maintaining JSDocs. They always eventually get out of sync with the code.

Maybe we could use Flow here instead?